test/main.js

```diff
@@ -178,7 +178,7 @@ const createMocha = (driver, testCase) => {
     )
   })
   mocha.suite.beforeEach('Set retry', function () {
-    this.currentTest.retries(2)
+    this.currentTest.retries(1)
```

Comment:
Is this going to speed up tests feedback?

Reply:
Well, I noticed that we don't need to try 2 additional times. Our main issues on retires at the moment is this general 'Async....' errors.